lang/java/avro/src/main/java/org/apache/avro/SchemaCompatibility.java

```diff
@@ -432,26 +433,20 @@ private SchemaCompatibilityType calculateCompatibility(
     /** Validated writer schema. */
     private final Schema mWriter;
 
-    /** Human readable description of this result. */
-    private final String mDescription;
-
     /**
      * Constructs a new instance.
      *
      * @param type of the schema compatibility.
      * @param reader schema that was validated.
      * @param writer schema that was validated.
-     * @param description of this compatibility result.
      */
     public SchemaPairCompatibility(
```

Comment:
Since this is part of the public API, this constructor should still be supported, but possibly deprecated.